src/metadata.rs

```diff
@@ -61,13 +61,16 @@ pub fn load(datadir: &Path) -> eyre::Result<PostMetadata> {
 }
 
 #[repr(C)]
-#[derive(Debug, Clone)]
+#[serde_as]
+#[derive(Debug, Clone, Deserialize, Serialize)]
 pub struct ProofMetadata {
+    #[serde_as(as = "Base64")]
     pub node_id: [u8; 32],
+    #[serde_as(as = "Base64")]
     pub commitment_atx_id: [u8; 32],
+    #[serde_as(as = "Base64")]
     pub challenge: [u8; 32],
     pub num_units: u32,
-    pub labels_per_unit: u64,
```

Comment:
Will this cause the value to also be dropped from the json file? We should check if that breaks the node internal initializer in the "supervised" setup.

Reply:
It won't break because I removed `labels_per_unit` from the **proof** metadata structure. The initializer uses a different structure for the **post** metadata.